packages/flutter_tools/test/commands.shard/hermetic/build_windows_test.dart

```diff
@@ -126,7 +130,7 @@ void main() {
   }
 
   testUsingContext('Windows build fails when there is no cmake path', () async {
-    final BuildWindowsCommand command = BuildWindowsCommand(logger: BufferLogger.test())
+    final BuildWindowsCommand command = BuildWindowsCommand(logger: BufferLogger.test(), operatingSystemUtils: globals.os)
```

Comment:
since this the path for this test includes "hermetic", this test should be hermetic and provide a faked out instance of `OperatingSystemUtils`

Reply:
Ok, thanks for explaining this. Updated those tests to use `FakeOperatingSystemUtils()` instead.